zulipterminal/cli/run.py

```diff
@@ -181,11 +185,13 @@ def main(options: Optional[List[str]]=None) -> None:
     except ServerConnectionFailure as e:
         print(in_color('red',
                        "\nError connecting to Zulip server: {}.".format(e)))
+        logging.exception(e)
         sys.exit(1)
     except Exception as e:
+        logging.exception(e)
```

Comment:
This looks good, to resolve the issue I pointed at in the first commit. I would probably have moved this commit to before the first, or possibly squashed the two together

Also, if each traceback will append to this file, should we have a separator to distinguish between tracebacks from different events?